Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM python:2.7
+FROM python:2.7-alpine
 WORKDIR /opt/ftw
```

Comment:
Do you know what version of the bourne{again} shell is being installed with the Alpine distribution? It doesn't look like the arguments are being parsed/passed to the `build_journal.py` script from `docker_entry.sh` the same way.

Reply:
The problem appears to be the result of differences between the distribution's version of `mktemp`. Looking into it a bit more

Reply:
Don't waste too much time on it, i just PR'ed this without checking the actual runtime. I can have a closer look soon :)

Reply:
```diff
@@ -14,7 +14,7 @@ while getopts "Dd:f:F" opt; do
             if [ "$OPTARG" != "-" ]; then
                 ruledir=$OPTARG
             else
-                T=`mktemp -d /tmp/rules.XXXXX`
+                T=`mktemp -d -t rules.XXXXXX`
                 while IFS= read LINE; do
                     echo "$LINE" >> $T/rules.yaml
                 done
```